src/js/control-bar/time-controls/time-display.js

```diff
@@ -38,18 +38,16 @@ class TimeDisplay extends Component {
   createEl(plainName) {
     const className = this.buildCSSClass();
     const el = super.createEl('div', {
-      className: `${className} vjs-time-control vjs-control`
+      className: `${className} vjs-time-control vjs-control`,
+      innerHTML: `<span class="vjs-control-text">${this.localize(this.labelText_)}&nbsp;</span>`
     });
 
-    this.contentEl_ = Dom.createEl('div', {
+    this.contentEl_ = Dom.createEl('span', {
```

Comment:
this is probably fine to make, but I'd like to take a look at the output first.

Reply:
If we use a `<div>`, JAWS treats the content label and the time as two separate things, so you need to press the arrow key to move between them in Virtual Cursor mode. With a `<span>`, JAWS treats them as a single thing (hence the need to be careful about adding a space in certain places with spans).

Reply:
yeah, I don't see any issues with this change once trying it out.